state/state.go

```diff
@@ -2082,33 +2086,14 @@ type ControllerInfo struct {
 	// MachineIds holds the ids of all machines configured to run a controller.
 	// Check the individual machine docs to know if a given machine wants to vote and/or has the vote.
 	MachineIds []string
-
-	// MongoSpaceName is the space that contains all Mongo servers.
-	MongoSpaceName string
-
-	// MongoSpaceState records the state of the mongo space selection state machine. Valid states are:
-	// * We haven't looked for a Mongo space yet (MongoSpaceUnknown)
-	// * We have looked for a Mongo space, but we didn't find one (MongoSpaceInvalid)
-	// * We have looked for and found a Mongo space (MongoSpaceValid)
-	// * We didn't try to find a Mongo space because the provider doesn't support spaces (MongoSpaceUnsupported)
-	MongoSpaceState MongoSpaceStates
```

Comment:
I think this is correct, but we should be careful if this was in an API that we want to preserve compatibility. Offhand I don't think we need to, but we should audit for it.

Reply:
As of now, these are only referred to in the upgrade and tests for it.